eng/packages/http-client-csharp/generator/Azure.Generator/src/Primitives/NewAzureProjectScaffolding.cs

```diff
@@ -23,10 +23,10 @@ protected override string GetSourceProjectFileContent()
         {
             var builder = new CSharpProjectWriter()
             {
-                Description = $"This is the {AzureClientPlugin.Instance.Configuration.RootNamespace} client library for developing .NET applications with rich experience.",
-                AssemblyTitle = $"SDK Code Generation {AzureClientPlugin.Instance.Configuration.RootNamespace}",
+                Description = $"This is the {AzureClientPlugin.Instance.InputLibrary.InputNamespace.Name} client library for developing .NET applications with rich experience.",
+                AssemblyTitle = $"SDK Code Generation {AzureClientPlugin.Instance.InputLibrary.InputNamespace.Name}",
                 Version = "1.0.0-beta.1",
-                PackageTags = AzureClientPlugin.Instance.Configuration.RootNamespace,
+                PackageTags = AzureClientPlugin.Instance.InputLibrary.InputNamespace.Name,
```

Comment:
it is correct that we use this value here?
I think here it should be the package name.
If we are not sure about this, please add a TODO and open an issue for it.

Reply:
right, it should be `AzureClientPlugin.Instance.TypeFactory.RootNamespace`, udpated.